test/test_dht.cpp

```diff
@@ -3480,13 +3512,13 @@ TORRENT_TEST(dht_state)
 {
 	dht_state s;
 
-	s.nid = to_hash("0000000000000000000000000000000000000001");
+	s.nids.emplace_back(address::from_string("1.1.1.1"), to_hash("0000000000000000000000000000000000000001"));
 	s.nodes.push_back(uep("1.1.1.1", 1));
 	s.nodes.push_back(uep("2.2.2.2", 2));
 	// not important that IPv6 is disabled here
-	s.nid6 = to_hash("0000000000000000000000000000000000000002");
-	s.nodes6.push_back(uep("3.3.3.3", 3));
```

Comment:
I had to remove those because they no longer work if the host system doesn't have IPv6 support. This can only be reliably tested in the simulator.